bindings/pyroot/pythonizations/test/rdf_define_pyz.py

```diff
@@ -0,0 +1,67 @@
+import unittest
```

Comment:
Also, please add at least one test where the input types of the python callable are columns that hold vectors and they are used as numpy arrays inside the python callable